metricbeat/module/system/diskio/_meta/data.json

```diff
@@ -1,45 +1,30 @@
 {
-    "@timestamp": "2016-05-23T08:05:34.853Z",
+    "@timestamp": "2017-10-12T08:05:34.853Z",
     "beat": {
         "hostname": "host.example.com",
         "name": "host.example.com"
     },
     "metricset": {
-        "host": "localhost",
         "module": "system",
         "name": "diskio",
         "rtt": 115
     },
     "system": {
         "diskio": {
             "io": {
-                "time": 4130
+                "time": 955214
             },
-            "name": "vda1",
+            "name": "disk0",
             "read": {
-                "bytes": 204800,
-                "count": 36,
-                "time": 960
+                "bytes": 49130488320,
+                "count": 4045827,
+                "time": 695534
             },
             "write": {
-                "bytes": 2281472,
-                "count": 224,
-                "time": 29700
-            },
-            "iostat": {
-                "avg_queue_size": 0.020000,
```

Comment:
It would be nice to advertise these metrics in the example. Is there a config option we can set in `TestData` or do you want only the default fields in the example?

Reply:
I would definitively like to have these in too. I think the problem here is that I did run this on my Mac where these values are not available. Let me do a run for this data set in a linux container.